daemon/stats.go

```diff
@@ -126,6 +129,20 @@ func (daemon *Daemon) ContainerStats(ctx context.Context, prefixOrName string, c
 			if !config.Stream {
 				return nil
 			}
+		case <-time.After(statsInterval):
```

Comment:
What's the difference? I don't see the big advantage over `time.After()` :smile: